From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 16 Dec 2021 15:48:23 -0400
Subject: Use sogs instead of python3 -msogs

For deb package we include a sogs script that changes to the appropriate
user to invoke the command; this patch rewrites the help messages to
reflect that.
---
 sogs/__main__.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/sogs/__main__.py b/sogs/__main__.py
index 3d36f6b..742505a 100644
--- a/sogs/__main__.py
+++ b/sogs/__main__.py
@@ -6,21 +6,22 @@ import sys
 from . import __version__ as version
 
 ap = AP(
+    prog="sogs",
     epilog="""
 
 Examples:
 
     # Add new room 'xyz':
-    python3 -msogs --add-room xyz --name 'XYZ Room'
+    sogs --add-room xyz --name 'XYZ Room'
 
     # Add 2 admins to each of rooms 'xyz' and 'abc':
-    python3 -msogs --rooms abc xyz --admin --add-moderators 050123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef 0500112233445566778899aabbccddeeff00112233445566778899aabbccddeeff
+    sogs --rooms abc xyz --admin --add-moderators 050123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef 0500112233445566778899aabbccddeeff00112233445566778899aabbccddeeff
 
      # Add a global moderator visible as a moderator of all rooms:
-    python3 -msogs --add-moderators 050123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef --rooms=+ --visible
+    sogs --add-moderators 050123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef --rooms=+ --visible
 
      # List room info:
-    python3 -msogs -L
+    sogs -L
 
 A sogs.ini will be loaded from the current directory, if one exists.  You can override this by
 specifying a path to the config file to load in the SOGS_CONFIG environment variable.
